tests/test_jsonparser.py

```diff
@@ -347,15 +370,5 @@ def test_zbox_label_roundtrip(self):
         self.assertEqual(g2.vdata(v1, 'name'), 'my vertex')
         self.assertEqual(g2.vdata(v2, 'custom'), 42)
 
-        js = json.dumps(d)
-        d2 = json.loads(js)
-        g3 = dict_to_graph(d2)
-
-        self.assertEqual(g3.vdata(v1, 'label'), Fraction(3, 4))
-        self.assertEqual(g3.vdata(v2, 'label'), poly_label)
-        self.assertEqual(g3.vdata(v3, 'label'), 2.5 + 1.3j)
-        self.assertEqual(g3.vdata(v1, 'name'), 'my vertex')
-        self.assertEqual(g3.vdata(v2, 'custom'), 42)
-
```

Comment:
The merge seems to have somehow deleted these lines (which are unrelated to this MR). I'll restore them.